### ISSUE TITLE
Detail settings for Wan2.1?

### ISSUE THREAD

**Issue by hkunzhe** [Vchitect/DCM#13]
I found the implementation details for HunyuanVideo from the paper:
> The distillation was conducted on 129-frame video sequences at a resolution of 1280 × 720 with a batch size of 6. For the semantic expert denoiser, we performed 1000 iterations of distillation with a learning rate of 1e−6, while for the details expert denoiser, we trained for 1000 iterations with a learning rate of 5e−6.

However, as in [distill_hy_detail_expert.sh](https://github.com/Vchitect/DCM/blob/main/scripts/distill/distill_hy_semantic_expert.sh), the total batch size is 3\*8\*1\*1=24 rather than 6.

What is the actual setup for Wan2.1 distillation (batch size/internal dataset size and etc.)?

**Comment by cszy98** [comment]
Since the sequence parallel size (sp_size) being set to 4, the actual batch size is not 24. Due to limited computational resources, we used a batch size of 4 for training WAN2.1.

**Comment by hkunzhe** [comment]
Thanks for the elaboration!
